src/functions/calculateTargetAllocation.ts

```diff
@@ -0,0 +1,70 @@
+import Allocation, { AssetTypes } from "src/interfaces/allocation"
+
+const EMPTY_TARGETS: Allocation[] = [
+  { type: "celo-native-asset" as const, token: "CELO", percent: 0 },
+  { type: "other-crypto-assets" as const, token: "BTC", percent: 0 },
+  { type: "other-crypto-assets" as const, token: "ETH", percent: 0 },
+  { type: "stable-value" as const, token: "DAI", percent: 0 },
+  { type: "stable-value" as const, token: "USDC", percent: 0 },
+  { type: "natural-capital" as const, token: "cMC02", percent: 0 },
+]
+
+export function calculateTargetAllocation(
+  outstandingStablesUSD: number,
+  totalReserveUSD: number
+): Allocation[] {
+  const targetAllocation: Allocation[] = EMPTY_TARGETS
+
+  let celoTarget: number
+  let stablesTarget: number
+  let natCapTarget: number
+  let otherCryptoTarget: number
+
+  if (totalReserveUSD / outstandingStablesUSD >= 2) {
+    stablesTarget = outstandingStablesUSD / totalReserveUSD
+    celoTarget = 0.5
+    otherCryptoTarget = 1 - stablesTarget - celoTarget
+    natCapTarget = (1 - stablesTarget - celoTarget) * 0.02
+  } else if (totalReserveUSD / outstandingStablesUSD < 2) {
```

Comment:
`&& totalReserveUSD / outstandingStablesUSD >= 1`?

Reply:
I'm not sure about that. The cases for this block reflect the logic in [CGP 62](https://celo.stake.id/#/proposal/62)

What are you thinking?

Reply:
That if `totalReserveUSD / outstandingStablesUSD` is less than 1 it'll never reach the second `else if` (because it matched the first one).

Reply:
Ah i see. I've added that condition.